index.mako

```diff
@@ -89,6 +89,84 @@ Using [conda](https://conda.io) (via [conda-forge](https://conda-forge.org)):
 conda install -c conda-forge pre_commit
 ```
 
+## Quick start
+
+### 1. Install
+
+- follow the [install](#install) instructions above
+- `pre-commit --version` should show you what version you're using
+
+```console
+$ pre-commit --version
+pre-commit 1.18.1
+```
```

Comment:
one thing about this section is it's not entirely clear that you're installing the tool here for yourself, not into the repo

maybe just changing the heading to `1. Install pre-commit` would make that clear?

Reply:
great suggestion! fixed :)